app/src/main/java/com/keylesspalace/tusky/entity/Account.kt

```diff
@@ -36,8 +36,8 @@ data class Account(
     val bot: Boolean = false,
     val emojis: List<Emoji>? = emptyList(), // nullable for backward compatibility
     val fields: List<Field>? = emptyList(), // nullable for backward compatibility
-    val moved: Account? = null
-
+    val moved: Account? = null,
+    val roles: List<Role>? = emptyList()
```

Comment:
Is this somehow cached somewhere (or is this only AccountEntity)? So could there then be accounts without roles which have some?

Reply:
Data for AccountActivity is always freshly loaded, no cache.